### DISCUSSION TITLE
ERROR: Make sure your audio length has enough samples for a single feature in 2nd epochs

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#6723] [Q&A]
I'm getting this error. In the middle of `2nd epoch`. Could not find any clue why this happening in 2nd epoch.
If there was a problem with my data then it should occure in my 1st epoch. But it's happening in 2nd epoch
```
ValueError: normalize_batch with `per_feature` normalize_type received a tensor of length 1. This will result in torch.std() returning nan. Make sure your audio length has enough samples for a single feature (ex. at least `hop_length` for Mel Spectrograms)
```

**Comment by mehadi92** [comment]
There was 1 corrupted audio files in my dataset

**Reply by AudranBert** [reply]
It could be great if it gives the name of the file, because I cleaned my data and still had the errors and I don't know where it comes from

**Reply by mehadi92** [reply]
@AudranBert,  I use a custom dataset. In my dataset, there were some 0-length audio files
